dash/__init__.py

```diff
@@ -20,6 +20,7 @@
 from .version import __version__  # noqa: F401,E402
 from ._callback_context import callback_context  # noqa: F401,E402
 from ._callback import callback, clientside_callback  # noqa: F401,E402
+from ._callback import NoUpdate as NoUpdateType  # noqa: F401,E402
```

Comment:
Might be better do rename it inside `_callback.py`, otherwise there might be situations where the same thing shows up as either name. I wouldn't consider that a breaking change, this hasn't ever been part of the exposed API.

Reply:
@KoolADE85 brought up this point as well.

Compare:
```python
type(no_update) # <class 'dash._callback.NoUpdate'>
```
whereas with `None`, it's
```python
type(None)      # <class 'NoneType'>
```

We'll want this `type` to be discoverable & renaming rather than aliasing enables that.

Reply:
I'd rather it not be postfixed.

Reply:
@T4rk1n how come?

Reply:
What we have here is not a type but a class. Since everything is a class in Python it feel meaningless to have a postfix of `Type` which it is not.